File: EDMTDev/5_Zooming_ImageView/app/src/main/java/com/rasyidcode/a5_zooming_imageview/MainActivity.java
package com.rasyidcode.a5_zooming_imageview;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ImageButton;
import android.widget.ImageView;

import com.github.chrisbanes.photoview.PhotoView;
import com.github.chrisbanes.photoview.PhotoViewAttacher;

public class MainActivity extends AppCompatActivity {

//    private ImageView mImageView;
    private PhotoView mPhotoView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

//        mImageView = findViewById(R.id.image_pokimane);
//        PhotoViewAttacher photoViewAttacher = new PhotoViewAttacher(mImageView);
//        photoViewAttacher.update();

        mPhotoView = findViewById(R.id.photo_view_pokimane);
    }
}

File: from-article/codelabs-tuts/OnePointThree_Challenge/app/src/main/java/me/jamilalrasyidis/onepointthree_challenge/MainActivity.kt
package me.jamilalrasyidis.onepointthree_challenge

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.ImageView
import androidx.core.content.ContextCompat
import kotlin.random.Random

class MainActivity : AppCompatActivity() {

    private lateinit var diceImageLeft: ImageView
    private lateinit var diceImageRight: ImageView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btnRollDice: Button = findViewById(R.id.btn_roll_dice)
        btnRollDice.setOnClickListener { rollDice() }

        val btnClearDice: Button = findViewById(R.id.btn_clear_dice)
        btnClearDice.setOnClickListener { clearDice() }

        diceImageLeft = findViewById(R.id.iv_dice_image_left)
        diceImageRight = findViewById(R.id.iv_dice_image_right)
    }

    private fun rollDice() {
        diceImageLeft.setImageResource(getRandomDiceImage())
        diceImageRight.setImageResource(getRandomDiceImage())
    }

    private fun clearDice() {
        diceImageLeft.setImageResource(R.drawable.empty_dice)
        diceImageRight.setImageResource(R.drawable.empty_dice)
    }

    private fun getRandomDiceImage(): Int {
        return when (Random.nextInt(6) + 1) {
            1 -> R.drawable.dice_1
            2 -> R.drawable.dice_2
            3 -> R.drawable.dice_3
            4 -> R.drawable.dice_4
            5 -> R.drawable.dice_5
            else -> R.drawable.dice_6
        }
    }
}
